docs/docs/chitchat-faqs.mdx

```diff
@@ -2,336 +2,72 @@
-```
-
-:::note Lots of intents
-Do you have lots of intents that you want to respond to? In that case you should
-check out [Handling FAQs](./chitchat-faqs.mdx#handling-faqs-using-a-response-selector).
-You'll avoid the need to specify one story for each of your intents.
-:::
-
-You'll need to tie all these pieces together by adding the intents, actions
-and responses to our [domain](domain.mdx):
-
-```yaml-rasa title="domain.yml"
-intents:
-- greet
-- bye
-- thank
-
-responses:
-  utter_noworries:
-  - text: No worries!
-
-  utter_greet:
-  - text: Hi
-
-  utter_bye:
-  - text: Bye!
-```
-
-### 3. Using the bot
-You can now train a first model and test the bot, by running the following commands:
-
-```bash
-rasa train
-rasa shell
-```
-
-This bot should now be able to reply to the intents we defined consistently, and in any order.
-
-For example:
-
-<img alt="Memoization Policy Conversation" src={useBaseUrl("/img/memoization_policy_convo.png")} />
-
-### 4. Testing the bot
-
-While it's good to test the bot interactively, you should also add story tests that
-can later be included as part of a [CI/CD system](./setting-up-ci-cd).
-[Story tests](./testing-your-assistant#end-to-end-testing)
-include user messages, bot actions and responses. This ensures that
-the dialogue handling is tested as well as the message handling.
-The initial project already contains test conversations, you can replace
-them with some test conversations for your assistant:
-
-```yaml-rasa title="tests/test_stories.yml"
-stories:
-- story: greet and goodybe
-  steps:
-  - user: |
-     Hi!
-    intent: greet
-  - action: utter_greet
-  - user: |
-     Bye
-    intent: bye
-  - action: utter_bye
-
-- story: greet and thanks
-  steps:
-  - user: |
-     Hello there
-    intent: greet
-  - action: utter_greet
-  - user: |
-     thanks a bunch
-    intent: thank
-  - action: utter_noworries
-
-- story: greet and thanks and goodbye
-  steps:
-  - user: |
-     Hey
-    intent: greet
-  - action: utter_greet
-  - user: |
-     thanks you
-    intent: thank
-  - action: utter_noworries
-  - user: |
-     bye bye
-    intent: bye
-  - action: utter_bye
-```
-
-This test file contains three separate test stories. You can
-test your bot on all of them using `rasa test`:
-
-```bash
-rasa test --stories tests/test_stories.yml
+# other policies
+- name: RulePolicy
 ```
 
-The test command will produce a directory named `results`. It will contain a file
-called `failed_stories.yml`, where any test cases that failed will be printed. It will
-also specify whether it was an NLU or Core prediction that went wrong.  As part of a
-CI/CD pipeline, the test option `--fail-on-prediction-errors` can be used to throw
-an exception that stops the pipeline:
+The `RulePolicy` works alongside other policies in your configuration (like the `TEDPolicy`). Its job is to enforce rule-based logic for conversation paths where you already know how the assistant should respond to a given input.
```

Comment:
hm, not sure i would say that "rule-based logic for conversation paths where you already know how the assistant should respond to a given input" - arguably you always know what the assistant should respond.
Other question - do we think we need to explain how the rule policy functions here? (not necessarily saying we shouldn't, just asking for your opinion

Reply:
From what we talked about re. keeping reasons for using and not explanations, no, I think we don't need to